### PR TITLE
Cloud changelog - Dec 8

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -3,12 +3,38 @@
...
-- [Release 2021-10-13](#release-2021-10-13)
+
+## Release 2021-12-08
+
+### Improvements
+
+#### User Interface (UI)
+ - Added thread replies to search results when collapsed reply threads feature is enabled.
+ - Updated "Terms of Service" terminology to "Terms of Use" product-wide.
+ - Government related references are now shown for Government type licenses.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:20]:
```suggestion
 - Government-related references are now shown for Government type licenses.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -3,12 +3,38 @@
...
+
+#### Performance
+ - Added a general performance fix for loading the web application and typing.
+ - Improved performance while typing by moving some autocomplete layout calculation.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:24]:
```suggestion
 - Improved performance while typing by moving some autocomplete layout calculations.
```

Comment (cwarnermm) [reply source/install/cloud-changelog.md:24]: I'm not 100% sure that pluralizing the word "calculations" is technical accurate (where multiple calculations were impacted by a change); I pluralized it to make the sentence flow a bit better.

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -3,12 +3,38 @@
...
+
+### Bug Fixes
+ - Fixed slow channel loading for instances with website link previews enabled.
+ - Fixed an issue where the webapp crashed sometimes when clicking on an image file from "Recent files".
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:28]:
```suggestion
 - Fixed an issue where the webapp crashed sometimes when selecting an image file from "Recent files".
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/install/cloud-changelog.md]: Removed it as this is more relevant to on-prem.

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -3,12 +3,41 @@
...
 - [Release 2021-11-10](#release-2021-11-10)
 - [Release 2021-10-27](#release-2021-10-27)
-- [Release 2021-10-13](#release-2021-10-13)
+
+## Release 2021-12-08
+
+### Improvements
+
+#### User Interface (UI)
+ - Added thread replies to search results when collapsed reply threads feature is enabled.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:18]:
```suggestion
 - Added thread replies to search results when Collapsed Reply Threads is enabled.
```

**Review by cwarnermm — COMMENTED** [review]
Thanks! Feedback provided inline.

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -3,12 +3,42 @@
...
+
+#### User Interface (UI)
+ - Added thread replies to search results when Collapsed Reply Threads is enabled.
+ - Updated "Terms of Service" terminology to "Terms of Use" product-wide.
+
+#### Performance
+ - Added a general performance fix for loading the web application and typing.
+ - Improved performance while typing by moving some autocomplete layout calculations.
+ - Improved performance by reducing DOM usage during render.
+ - Removed real-time updates of a couple features to prevent overloading servers on user updates.
```
[the remark this answers is not in the document]
Comment (cwarnermm) [reply source/install/cloud-changelog.md:25]: We may want to consider specifying which features no longer have real-time updates for clarity and transparency.

**Review by cwarnermm — APPROVED** [review]
Provided non-blocking feedback inline.

File: source/install/cloud-changelog.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (amyblais) [reply source/install/cloud-changelog.md:25]: cc @hmhealey if you have more details on this.

**Review by marianunez — APPROVED** [review]

**Review by hmhealey — APPROVED** [review]

**Pull request merged**